core/src/main/java/org/mobilitydata/gtfsvalidator/parsing/RowParser.java

```diff
@@ -416,7 +416,7 @@ public ValueParser(String formatName) {
 
         final T parseField(int columnIndex, boolean required) {
             String s = asString(columnIndex, required);
-            if (s == null || s.isEmpty()) {
```

Comment:
Could Guava [isNullOrEmpty](https://guava.dev/releases/27.1-jre/api/docs/com/google/common/base/Strings.html#isNullOrEmpty-java.lang.String-) be used?

Reply:
We never have an empty string anymore, so I just compare it with `null`.